contracts/programs/ocr2/src/context.rs

```diff
@@ -79,8 +79,7 @@ pub struct Transmit<'info> {
     #[account(mut, address = state.load()?.transmissions)]
     pub transmissions: Account<'info, Transmissions>,
 
-    #[account(address = store::ID)]
-    pub store_program: AccountInfo<'info>,
```

Comment:
This is the bulk of the change. The rest just adds the `Store` type and renames the old `Store` type to `State`.